financialaid/templatetags/financial_aid_tags.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
I would prefer moving this to the views to make the code cleaner. Templates should be relatively simple and computation and lookups are generally cleaner when written in Python rather than the template syntax